### PR TITLE
Add county locations 

### PR DESCRIPTION
**Pull request by ellisrichardj**
The county location markers are fixed (essentially a point at the approximate centre of the area.  Some of the 'counties' are regions used as part of the CPH system, so may not exactly correspond to administrative counties

### CODE REVIEW

**Review by nick-pestell — APPROVED** [review]
I think this is fine. I thought you already had this in tbh. Maybe it's just moving it to `accessory`?

**Comment by ellisrichardj** [comment]
> I think this is fine. I thought you already had this in tbh. Maybe it's just moving it to `accessory`?

Yes - just moving it into the repo as its static

File: accessory/counties.tsv [hunk 1]
```diff
@@ -0,0 +1,86 @@
+LocationType	County	Lat	Long
+County	Aberdeenshire	57.26782	-2.552379
```
Comment (afishman) [inline accessory/counties.tsv:2]: Nice work! Great to see this go in ;)

**Review by afishman — COMMENTED** [review]
@ @

**Pull request merged**